### DISCUSSION TITLE
以後會新增點技能功能嗎

### DISCUSSION THREAD

**Discussion by Ginsaiyan** [NateScarlet/auto-derby#153] [Ideas]
可以像賽事依樣做個技能欄 讓他自己點技能 可以更自動得感覺

**Comment by NateScarlet** [comment]
现在的手动加技能点对我够用了 因为我不刷历战

有人自己实现了
https://github.com/yourlies/auto-derby/blob/130d9c186d13216f81da4367130326a101eb5432/auto_derby/single_mode/skill.py

不过不支持插件设置 而且基于的版本比较老

**Reply by NateScarlet** [reply]
去 #155 投票

人多了再考虑做
